spring-security-oauth-server/src/main/java/org/baeldung/mvc/config/EmployeeController.java

```diff
@@ -0,0 +1,35 @@
+package org.baeldung.mvc.config;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.springframework.http.HttpStatus;
+import org.baeldung.mvc.model.Employee;
+import org.springframework.http.MediaType;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@Controller
+public class EmployeeController {
+
+    private List<Employee> employees = new ArrayList<>();
+
+    @RequestMapping(value = "/employee")
+    @ResponseBody
+    public Employee getEmployee(@RequestParam String email) {
+        Employee result = employees.stream().filter(x -> email.equals(x.getEmail())).findAny().orElse(null);
```

Comment:
"email.equals(x.getEmail())" we might want to reverse it, email can be null, right?